app/code/Magento/Catalog/Controller/Adminhtml/Product/SuggestAttributeSets.php

```diff
@@ -46,4 +46,15 @@ public function execute()
         );
         return $resultJson;
     }
+    
+    /**
+     * Check for is allowed
+     *
+     * @return boolean
+     */
+    protected function _isAllowed()
```

Comment:
Listed files have been fixed already in https://github.com/magento/magento2/commit/148bae13d759f133a793723ff9bcd2b55ac7ebc2.
Unfortunately we still have some legacy code that require refactoring and will appreciate your help in this.